cmd/kola/kola.go

```diff
@@ -45,7 +45,12 @@ var (
 	cmdRun = &cobra.Command{
 		Use:    "run [glob pattern]",
 		Short:  "Run kola tests by category",
-		Long:   "run all kola tests (default) or related groups",
+		Long:   `Run all kola tests (default) or related groups.
+
+If the glob pattern is exactly equal to the name of a single test, any
+restrictions on the versions of Container Linux supported by that test
+will be ignored.
```

Comment:
Is it only the `MinVersion` & `MaxVersion` traits that are ignored or will it also ignore the `ExcludedPlatform` trait?